FibonacciNumbers.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class FibonacciNumbers {
+  public static void main(String[] args) {
+    int max = 2147483647, num1 = 0, num2 = 1, sum = 0, a = 0;
+    boolean ind1 = false, ind2 = false;
+    Scanner in = new Scanner (System.in);
+    try {
+      while (ind1 == false) {
+        System.out.println("Enter a number");
+        a = in.nextInt();
+        if (a < 0) {
+          System.out.println("Your number is less than zero. Repeat, please");
+        } else {
+          ind1 = true;
+        }
+      }
+    } catch (Exception e) {
+        System.out.println("You did not enter a number or your number is very large. The program stops working");

```

Comment:
Why do you allow to re-enter value for negative case and don't allow if it's none-integer?